autodocodec/src/Autodocodec/Codec.hs

```diff
@@ -1682,6 +1698,18 @@ orNullHelper = dimapCodec f g
       Nothing -> Nothing
       Just a -> Just (Just a)
 
+collapseVoidEither
```

Comment:
I want the schema description for `Either x Void` and `Either Void x` to be identical to `x` - this is my attempt at working towards that.